src/odemis/gui/test/comp_text_test.py

```diff
@@ -88,13 +90,22 @@ def test_unit_float(self):
 class NumberTextCtrlTestCase(test.GuiTestCase):
     frame_class = test.test_gui.xrcbutton_frame
 
+    def _check_wxpython_41(self):
+        """
+        Skip the test if wxpython < 4.1.
+        """
+        # UIActionSimulator.Text() doesn't seem to work on wxPython 4.0.7,
+        # but it works again on wxPython 4.1.
+        # Note: on wxPython 4.0.7, this works well within Eclipse, but not from a terminal.
+        wx_ver = tuple(int(v) for v in wx.__version__.split("."))
+        if wx_ver < (4, 1, 0):
+            self.skipTest("Test case is known to fail on wxPython < 4.1 due to buggy UIActionSimulator")
+
     def test_int_txt_ctrl(self):
 
         ctrl = IntegerTextCtrl(self.panel, value=123456789)
         self.add_control(ctrl, label=ctrl.__class__.__name__, flags=wx.EXPAND | wx.ALL)
 
-        test.goto_manual()
```

Comment:
why did you remove this?

Reply:
It's already done for the whole file at line 35. So this line has no effect.
Actually, I think we could probably change the `gui.test` so that this function is automatically done whenever it's imported (it just read an environment variable). But that's for another PR.